.github/pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+# Description
+
+Please include a summary of changes, motivation and context for this PR.
+
+Fixes # (issue)
+
+## Type of change
+
+Please tick all options which are relevant.
+
+- [ ] Bug fix
+- [ ] New feature
+- [ ] Added/removed dependencies
+- [ ] Requires documentation updates
+
+# Testing
+
+Please describe the test(s) that you added and ran to verify your changes.
+
+## Test(s) description
+
+**Test Configuration**:
+
+* OS:
+
+# Checklist:
+
+- [ ] I have added tests that prove my fix is effective and/or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] I have used understandable variable names
+- [ ] I have squashed my commits into a single commit ready for review
```

Comment:
This can be done when merging. Squashing during the review process makes it harder, because you cannot easily see what has changed since the last review only.